### ISSUE TITLE
Extend mungebits2 examples

### ISSUE THREAD

**Issue by frenchja** [syberia/mungebits2#53]
Should we extend the [mungebits examples](https://github.com/syberia/mungebits2#examples) to cover common analyses?  For example, these are the mungebits typically necessary for a a) [GBM](https://en.wikipedia.org/wiki/Gradient_boosting) or a b) multiple regression. This might enhance the adoption by lowering the barrier to usage while still allowing for developer customization.

We could also talk about organizing the various Syberia examples into a [Gitbook](https://www.gitbook.com/) that people can export and read.

https://github.com/syberia/mungebits2#examples

**Comment by robertzk** [comment]
@frenchja Definitely. Mungebits in particular would benefit from several vignettes illustrating progressively more complex usage (basic, simple wide transformations like imputation and sure independency screening, more complex wide column transformations, etc.).

What do you have in mind? Let's discuss during our next hackathon.
